httpstan/views.py

```diff
@@ -320,7 +324,8 @@ def logger_callback(operation: dict, message: bytes) -> None:
         # Deserializing it would be costly and require importing the protobuf Python module.
         if b"info:Iteration" not in message:
             return
-        operation["metadata"]["progress"] = message.split(b"info:", 1)[1].decode()
+        # When sampling completes rapidly, multiple iteration messages can be passed together. Return the final one.
+        operation["metadata"]["progress"] = iteration_info_re.findall(message).pop().decode()
```

Comment:
What happens if no progress info is given? Will this fail?
I think we don't have too long texts ro be parsed, so we don't need to optimize this. (reverse str and pattern, finditer+break (and reverse founds info))

Reply:
We are guaranteed to have Iteration information due to the `if` statement on line 321.